src/notation/view/widgets/editstyle.cpp

```diff
@@ -1336,7 +1340,8 @@ QString EditStyle::pageCodeForElement(const EngravingItem* element)
         return "page";
 
     case ElementType::INSTRUMENT_NAME:
-    case ElementType::TEXT: {
+    case ElementType::TEXT:
+    case ElementType::HARP_DIAGRAM: {
```

Comment:
Right click style menu on harp diagram now directs to `Text styles > Harp pedal diagram/Harp pedal text diagram`
 

https://github.com/musescore/MuseScore/assets/105774228/83183317-e029-4ae4-b8a6-5d9db3cc79ea

Reply:
Also now all the sub page codes and page codes are synced